Clover/app/src/main/java/org/floens/chan/ui/controller/ImageViewerController.java

```diff
@@ -263,8 +284,8 @@ public void setPagerItems(List<PostImage> images, int initialIndex) {
         pager.setCurrentItem(initialIndex);
     }
 
-    public void setImageMode(PostImage postImage, MultiImageView.Mode mode) {
-        ((ImageViewerAdapter) pager.getAdapter()).setMode(postImage, mode);
+    public void setImageMode(PostImage postImage, MultiImageView.Mode mode, boolean center) {
```

Comment:
if we're defaulting to true in most cases, and false is a special case, it might be helpful to create an overloaded method:
```
     public void setImageMode(PostImage postImage, MultiImageView.Mode mode) { 
         ((ImageViewerAdapter) pager.getAdapter()).setMode(postImage, mode, true); 
     } 
```
so you don't have to pass `true` each time

Reply:
This is a method in an implemented callback interface, so making an overloaded method wouldn't be any more or less complex than it is.